geoapps/inversion/magnetotellurics/constants.py

```diff
@@ -99,7 +99,7 @@
     "y_norm": 2.0,
     "z_norm": 2.0,
     "reference_model_object": None,
-    "reference_model": None,
+    "reference_model": 0.0,
```

Comment:
Should default to a reasonable conductivity. Let's say 1e-3 S/m

Reply:
done